### PR TITLE
fix leaking sw_vers

### PR DESCRIPTION
**Pull request by jetersen**
### Checklist
- [x] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass
- [x] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid
- [x] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I've updated the documentation if necessary.

### Motivation and Context

`system('sw_vers')` was leaking output:
```
ProductName:	Mac OS X
ProductVersion:	10.13
BuildVersion:	17A405
```

Simply running `fastlane` shows `sw_vers` info...
### Captured Output

Command Used: `fastlane`
<details><summary>Output/Log</summary>

```
                                                                                                                                                                                                    
[13:08:52]: Get started using a Gemfile for fastlane https://docs.fastlane.tools/getting-started/ios/setup/#use-a-gemfile
ProductName:	Mac OS X
ProductVersion:	10.13
BuildVersion:	17A405
[13:08:54]: Welcome to fastlane! Here's what your app is setup to do:
+--------+-----------------+------------------------------+
|                 Available lanes to run                  |
+--------+-----------------+------------------------------+
| Number | Lane Name       | Description                  |
+--------+-----------------+------------------------------+
| 1      | devices         |                              |
| 0      | cancel          | No selection, exit fastlane! |
+--------+-----------------+------------------------------+
[13:08:54]: Which number would you like run?
0

[!] Run `fastlane` the next time you need to build, test or release your app 🚀


```

</details>

### CODE REVIEW

File: fastlane_core/lib/fastlane_core/analytics/analytics_session.rb [hunk 1]
```diff
@@ -191,7 +191,7 @@ def operating_system_version
       os = self.operating_system
       case os
       when "macOS"
-        return system('sw_vers') ? `sw_vers -productVersion`.strip : 'unknown'
+        return system('sw_vers', out: File::NULL) ? `sw_vers -productVersion`.strip : 'unknown'
```
Comment (casz) [inline fastlane_core/lib/fastlane_core/analytics/analytics_session.rb]: @ohayon please take a look 👍

**Comment by KrauseFx** [comment]
Hey @casz, thanks for the PR, I noticed that also when using fastlane, do you know when this was introduced? Is that because of a new Xcode version?

**Comment by casz** [comment]
@KrauseFx nah it was introduced in #10625 :)

**Comment by mpirri** [comment]
@KrauseFx This was added as part of #10625 (not yet shipped to production).

**Review by mpirri — APPROVED** [review]
Thanks for catching this, @casz ! 💯

**Pull request merged**